test/kompo_ignore_test.rb

```
@@ -0,0 +1,127 @@
+require_relative "../lib/kompo"
```

RuboCop (1.82.1)

[convention] Missing frozen string literal comment.

(Style/FrozenStringLiteralComment)